Formula/cloudflare-access.rb

```diff
@@ -1,8 +1,8 @@
-class Cloudflared < Formula
-  desc "Cloudflare Tunnel client (formerly Argo Tunnel)"
+class CloudflareAccess < Formula
+  desc "Client for Cloudflare Access, Tunnel, and other Zero Trust products"
```

Comment:
Since the name of the software is still the same I don't think we'll want to rename this.

Reply:
I guess I'm not super familiar with what the class name is for - what are the cons of renaming it?

Reply:
The cons are:
It's no longer installable with the name of the software since it's been renamed to something else.

Reply:
That was the point of this PR - it's interfering with our internal tap, and since having this formula in homebrew-core is very new (1w old or so) I figured it wouldn't be very disruptive to change its name in homebrew-core.

Reply:
But it would mean that people who hear about `Cloudflared` can't find it in Homebrew/core anymore. Wouldn't the effort be better spent auto-updating the core version instead of making the product harder to find for non-internal users?

Reply:
Possibly - is there any infrastructure already in-place for auto-updating the core version?

Reply:
`brew bump` can be used for that. Or if you want more control over the parameters, `brew bump-formula-pr`